### PR TITLE
Proposed change to close button

### PR DESCRIPTION
**Pull request by wilsonanyonga**
I have changed the close button and removed the inkwell in favour of IconButton widget. this widget is more responsive on mobile compared to the inkwell button. I have also reduced the space between the top and the title row.

### CODE REVIEW

**Comment by mhmzdev** [comment]
@wilsonanyonga can you share the SS please?
Plus, if you have replaced the icon-button just to be focused on mobile this package is being used on web as well by a lot of people so won't it be causing any trouble there?

**Comment by wilsonanyonga** [comment]
@mhmzdev the change works well on the web as well. it just increases the clickable area.

WhatsApp Image 2023-11-26 at 21 33 16

**Review by wilsonanyonga — COMMENTED** [review]
Those are the suggested changes

File: lib/src/awesome_snackbar_content.dart [hunk 1]
```diff
@@ -178,21 +178,39 @@ class AwesomeSnackbarContent extends StatelessWidget {
                       ),
                     ),
 
-                    InkWell(
-                      onTap: () {
+                    // InkWell(
```
Comment (mhmzdev) [inline lib/src/awesome_snackbar_content.dart:181]: Please remove the commented code from here

**Review by mhmzdev — CHANGES_REQUESTED** [review]

**Review by mhmzdev — APPROVED** [review]

**Pull request merged**